base/operators.jl

```diff
@@ -279,8 +281,9 @@ false
 const ≠ = !=
 
 """
-    ===(x,y) -> Bool
-    ≡(x,y) -> Bool
+    ===(x, y) -> Bool
+    ≡(x, y) -> Bool
```

Comment:
I think it would be better to leave `≡` user-defined.

Reply:
Hum, it is quite old, but I think I don’t have any strong opinion about this symbol, so if you or others think it is better to leave it user-defined because it can have various meanings depending on the field, feel free to remove this part!

Reply:
@stevengj, hum, `const ≡ = ===` was written by Jeff Bezanson 6 years ago, I just added under `const ⩶ = ===`…

So, is there any advice for something elsewhere?

Reply:
Oh, I see, this was not your change.